### PR TITLE
Adding the online filtering and removing online from the map

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: tools/geoCodes.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const options = {
 
 const geocoder = NodeGeocoder(options);
 
-const locations = Array.from(new Set(events.map((event) => event.location.replace(" & Online", "")))).filter((location) => !cachedGeolocations[location]).filter((location) => location !== "").sort();
+const locations = Array.from(new Set(events.map((event) => event.location.replace(" & Online", "")))).filter((location) => !cachedGeolocations[location]).filter((location) => location !== "" && location !== "Online").sort();
```
Comment (alvinometric) [inline tools/geoCodes.js:15]: You may want to do `&& location.toLowerCase() !== "online"` to account for casing

Comment (jespino) [reply tools/geoCodes.js:15]: Actually, my expectation is that every line in the Readme has the same format, so I expect that to be case sensitive, anyway, changing it to case insensitive is a bit more reslient to accidental introduction of "online" string without uppercase in the readme.

Comment (jespino) [reply tools/geoCodes.js]: Done, thanks for the suggestion. :slightly_smiling_face:

**Comment by scraly** [comment]
Love it :)

**Pull request merged**